### PR TITLE
Add find_project_root (follow-up)

### PR DESCRIPTION
**Pull request by romasku**
This is follow-up PR to https://github.com/neuromation/platform-client-python/pull/1656

### CODE REVIEW

File: neuromation/api/utils.py [hunk 1]
```diff
@@ -112,3 +113,15 @@ async def retry() -> AsyncIterator[None]:
...
+
+
+def find_project_root(cwd: Optional[Path] = None) -> Path:
+    here = cwd if cwd is not None else Path.cwd()
+    while here.parent != here:
+        config = here / ".neuro.toml"
+        if config.exists():
+            return here
+        here = here.parent
+    from .config import ConfigError
```
Comment (romasku) [inline neuromation/api/utils.py:125]: We cannot avoid circular import here: `config.py` uses `find_project_root`, but it requires `ConfigError` to be imported.
Also I am still not sure that `ConfigError` is correct exeption here - I though it means "you config file is bad, delete it and make new one`, and here we just unable to find it at all.

Comment (romasku) [reply neuromation/api/utils.py:125]: Will be able to fix this when after https://github.com/neuromation/platform-client-python/pull/1657 is merged.

Comment (asvetlov) [reply neuromation/api/utils.py:125]: ok, let's keep the circular import.
`ConfigError` is probably good enough.
We can play building the comprehensive exceptions hierarchy game, but I prefer to do it on demand only.

File: neuromation/api/utils.py [hunk 2]
```diff
@@ -112,3 +115,13 @@ async def retry() -> AsyncIterator[None]:
 
 def flat(sql: str) -> str:
     return " ".join(line.strip() for line in sql.splitlines() if line.strip())
+
+
+def find_project_root(cwd: Optional[Path] = None) -> Path:
```
Comment (serhiy-storchaka) [inline neuromation/api/utils.py:120]: I am not sure `cwd` is good name.

Comment (romasku) [reply neuromation/api/utils.py]: I agree that `path` is more appropriate here, thanks for suggestion.

**Comment by asvetlov** [comment]
Please merge

**Pull request merged**